### PR TITLE
Update support libraries and implement material dialogs prelollipop

### PR DESCRIPTION
**Pull request by dabluck**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -35,7 +35,7 @@ def preDexEnabled = "true".equals(System.getProperty("pre-dex", "true"))
 
 android {
   compileSdkVersion 22
-  buildToolsVersion '22.0.0'
+  buildToolsVersion '22.0.1'
```
Comment (JakeWharton) [inline build.gradle]: You'll need to update `.travis.yml` with this value.

**Comment by JakeWharton** [comment]
LGTM. Just update `.travis.yml`. If you can, squash into one commit. If not, I'll do it when I merge.

**Comment by dabluck** [comment]
you're quick. did I do that right?

**Comment by JakeWharton** [comment]
You were super close. I'm not sure what happened but you managed to grab one of the commits already on `master` in your squash. I rebase your branch and merged it manually though.

Thanks!

**Pull request closed without merging**